store/tikv/gc_worker.go

```diff
@@ -67,7 +67,10 @@ func NewGCWorker(store kv.Storage) (*GCWorker, error) {
 	}
 	var ctx goctx.Context
 	ctx, worker.cancel = util.WithCancel(goctx.Background())
-	go worker.start(ctx)
+	var wg sync.WaitGroup
```

Comment:
Normally, `wg.Done()` is called when a goroutine exits. Seems to use a channel instead is better.